rosgraph_tui/controller.py

```diff
@@ -0,0 +1,210 @@
+import urwid
+from click import secho
+
+from rosnode import ROSNodeIOException
```

Comment:
Maybe we should guard the import to avoid ugly import errors if one didn't source a workspace yet.